packages/material-ui/src/TableContainer/TableContainer.js

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import clsx from 'clsx';
+import withStyles from '../styles/withStyles';
+
+export const styles = {
+  /* Styles applied to the root element. */
+  root: {
+    width: '100%',
+    overflowX: 'auto',
+  },
+};
+
+const TableContainer = React.forwardRef(function TableContainer(props, ref) {
+  const { classes, className, component: Component = 'div', ...other } = props;
+
+  return <Component ref={ref} className={clsx(classes.root, className)} {...other} />;
+});
+
+TableContainer.propTypes = {
+  /**
+   * The content of the table, normally `TableHead` and `TableBody`.
```

Comment:
Oops...

Reply:
I have just changed the wording based on the examples in the docs, please let me know if it is sufficient